firmware/firmware.ino

```diff
@@ -292,7 +329,8 @@ void setup()
   
   // Create secondary tasks for system.
   Scheduler.startLoop(motorUpdateLoop);
-  Scheduler.startLoop(serialConsoleLoop);
+  //Scheduler.startLoop(serialConsoleLoop);
```

Comment:
Unknown, we disabled a bunch of stuff once we noticed a delay in Qatar. Maybe make it enabled/disabled based on debug level?